internal/attestation/aws/snp/validator_test.go

```diff
@@ -63,84 +94,219 @@ func TestGeTrustedKey(t *testing.T) {
 	}
 }
 
-func TestTpmEnabled(t *testing.T) {
-	idDocNoTPM := imds.InstanceIdentityDocument{
-		ImageID: "ami-tpm-disabled",
-	}
-	userDataNoTPM, _ := json.Marshal(idDocNoTPM)
-	attDocNoTPM := vtpm.AttestationDocument{
-		InstanceInfo: userDataNoTPM,
-	}
+// TestValidateSNPReport has to setup the following to run ValidateSNPReport:
+// - parse ARK certificate from constants.go.
+// - parse cached ASK certificate.
+// - parse cached SNP report.
+// - parse cached AK hash. Hash and SNP report have to match.
+// - parse cache VLEK cert.
+func TestValidateSNPReport(t *testing.T) {
+	require := require.New(t)
+	certs, err := loadCerts(testdata.CertChain)
+	require.NoError(err)
+	ark := certs[1]
+	certs, err = loadCerts(testdata.CertChain)
+	require.NoError(err)
+	ask := certs[0]
```

Comment:
```suggestion
	ark := certs[1]
	ask := certs[0]
```
Or is there a reason for this?